### ISSUE TITLE
review missing curated diseases

### ISSUE THREAD

**Issue by Antonialock** [pombase/pombase-chado#773]
our import of 'curated diseases from other sources'  is out of date and slightly flawed.

once migration do->mondo is done:

a. quickfix
compare pombe:human 1:1 orthologs without a disease association against human genes with a disease association
(this was easy with old mondo website...not sure now...)

b. trickier-but-more-comprehensive fix 
compare pombe gene disease associations vs 1:1 ortholog disease association
(some genes will have more than one disease associated with them - we might have incompletely curated genes)

**Comment by ValWood** [comment]
i don't understand this:

>our import of 'curated diseases from other sources' is out of date and slightly flawed.

I understand that it can be out of date, but how else is it flawed?

>compare pombe:human 1:1 orthologs without a disease association against human genes with a disease association

why 1:1?

**Comment by ValWood** [comment]
> b. trickier-but-more-comprehensive fix
compare pombe gene disease associations vs 1:1 ortholog disease association
(some genes will have more than one disease associated with them - we might have incompletely curated genes)

I don't understand what is being  compared with what here

**Comment by Antonialock** [comment]
no that was badly described sorry.

What I mean is that since I last looked, many more diseases must have received disease associations (out of date).

When I last looked for missing diseases, I only looked for diseases for orthologs without ANY disease associations. some diseases will be associated with many diseases.

1:1 because if you have paralogs in pombe then either could perform the function of the human protein?

Basically, this ticket is about updating the disease associations based on existing curated resources.

**Comment by ValWood** [comment]
OK so basically we won't have everything because new stuff is added. So we need a robust way to periodically check other resources
Malacards, Monarch, etc to  pull in the new missing stuff?
Is that it?

**Comment by ValWood** [comment]
If so, this should transfer to the curation tracker for now
